docs/Deploying/Configuration.md

```diff
@@ -2522,3 +2522,27 @@ If not set then the Prometheus `/metrics` endpoint is disabled.
     basic_auth:
       password: "bar"
   ```
+
+#### Packaging
+
+Configurations used to control how Fleet interacts with the packaging server.
+These features are currently only intended to be used within Fleet Sandbox, but
+this is subject to change.
+
+##### packaging.global_enroll_secret
```

Comment:
We should indicate that the value should be kept secret and that we advice using randomly generated text of >= 24-bytes. (Or link to the method that we use in the code, `GenerateRandomText(24)`).

Reply:
awesome idea, I clarified this and included an example using `openssl`